src/webgpu/web-platform/copyImageBitmapToTexture.spec.ts

```diff
@@ -192,7 +232,7 @@ g.test('from canvas')
     // Generate original data.
     const imagePixels = new Uint8ClampedArray(bytesPerPixel * width * height);
     for (let i = 0; i < width * height * bytesPerPixel; ++i) {
-      imagePixels[i] = i % 256;
+      imagePixels[i] = i % 4 === 3 ? 255 : i % 256;
```

Comment:
nit: parentheses would be more readable
```suggestion
        imagePixels[i] = (i % 4 === 3) ? 255 : i % 256;
```

Reply:
Done.